packages/patternfly-4/react-core/src/components/Button/Button.md

```diff
@@ -39,15 +39,15 @@ import { Button } from '@patternfly/react-core';
 
 LinkButton = () => (
   <React.Fragment>
-    <Button component="a" href="https://pf-next.com/" target="_blank" variant="primary">
+    <Button component="a" href="//pf4.patternfly.org/" target="_blank" variant="primary">
```

Comment:
why the double slash at the beginning?

Reply:
Protocol relative links used to be preferred for portability (syncs http/https paths to that of the protocol on the consuming page) - a quick google tells me this is no longer preferred, I'll update.